types/db.ts

```diff
@@ -19,4 +19,5 @@ export interface Db {
     getStoreToken(storeId: string): Promise<string> | null;
     deleteStore(session: SessionProps): Promise<void>;
     deleteUser(session: SessionProps): Promise<void>;
+    hasAppExtensionsScope(storeHash: string): Promise<boolean> | boolean;
```

Comment:
I wonder if we should consider making this only `Promise<boolean>` and not `Promise<boolean> | boolean` to keep the interface consistent, so instead of returning `false` above on `mysql.ts` we should probably return a rejected promise instead?